proto/perp/v1/tx.proto

```diff
@@ -86,35 +86,32 @@ message MsgAddMarginResponse {
   Position position = 2;
 }
 
-// -------------------------- Liquidate --------------------------
-
-message MsgLiquidateResponse {
-  cosmos.base.v1beta1.Coin fee_to_liquidator = 1 [(gogoproto.nullable) = false];
-  cosmos.base.v1beta1.Coin fee_to_perp_ecosystem_fund = 2 [(gogoproto.nullable) = false];
-}
+// -------------------------- Liquidation --------------------------
 
 message MsgMultiLiquidate {
   string sender = 1;
 
-  message MultiLiquidation {
+  message SingleLiquidation {
     string pair = 1 [ 
       (gogoproto.customtype) = "github.com/NibiruChain/nibiru/x/common/asset.Pair",
       (gogoproto.nullable) = false
     ];
     string trader = 2;
   }
 
-  repeated MultiLiquidation liquidations = 2;
+  repeated SingleLiquidation liquidations = 2;
 }
 
 message MsgMultiLiquidateResponse {
-  message MultiLiquidateResponse {
-    oneof response {
-      string error = 1;
-      MsgLiquidateResponse liquidation = 2;
-    }
+  message SingleLiquidation {
```

Comment:
As this message is used for response, I think `LiquidationResponse` would be more meaningful.